test/unit_tests/calling/CallingRepositorySpec.js

```diff
@@ -68,19 +68,17 @@ describe('CallingRepository', () => {
 
   describe('startCall', () => {
     it('warns the user that there is an ongoing call before starting a new one', done => {
-      const activeCall = new Call(
-        selfUser.id,
-        createRandomUuid(),
-        CONV_TYPE.ONEONONE,
-        new Participant(),
-        CALL_TYPE.NORMAL,
-      );
+      const activeCall = new Call(selfUser.id, createRandomUuid(), CONV_TYPE.ONEONONE, new Participant(), 0, {
+        currentAvailableDeviceId: {
+          audioOutput: ko.pureComputed(() => 'test'),
+        },
+      });
       activeCall.state(CALL_STATE.MEDIA_ESTAB);
       spyOn(callingRepository.callState, 'activeCalls').and.returnValue([activeCall]);
       spyOn(amplify, 'publish').and.returnValue(undefined);
       const conversationId = createRandomUuid();
       const conversationType = CONV_TYPE.ONEONONE;
-      const callType = CALL_TYPE.NORMAL;
+      const callType = 0;
```

Comment:
Why not using an enum here?

Reply:
done in https://github.com/wireapp/wire-webapp/pull/10657/commits/74d3191599d112de83e13492cc0d84c1ea4db454